projects/bad_example/bad_example_fuzzer.cc

```diff
@@ -1,6 +1,17 @@
 // Copyright 2017 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+/* Copyright 2022 Google LLC
```

Comment:
I guess this is needed to get past our linter?

Reply:
yeah